dist/gcds-utility.css

```diff
@@ -25,11 +26,12 @@
   --gcds-color-green-500: #289f58; /* Must contrast 3:1 with white */
   --gcds-color-green-700: #03662a; /* Must contrast 7:1 with white */
   --gcds-color-purple-700: #7532b8; /* Must contrast 7:1 with white */
+  --gcds-color-orange-700: #be5a00; /* Must contrast 7:1 with white */
   --gcds-color-red-100: #fbddda;
   --gcds-color-red-500: #d3080c; /* Must contrast 3:1 with white */
   --gcds-color-red-700: #a62a1e; /* Must contrast 7:1 with white */
   --gcds-color-red-900: #822117;
-  --gcds-color-red-flag: #ff0000; /* Signature flag colour */
+  --gcds-color-red-flag: #eb2d37; /* Signature flag colour - FIP red */
   --gcds-color-yellow-100: #faedd1;
   --gcds-color-yellow-500: #b3800f; /* Must contrast 3:1 with white */
 }
```

Comment:
I'm not sure why these 3 tokens were missing / using a different colour in the previous version but I double checked those values with our token repo and those are the correct values that we should be using here.